everest-testing/src/everest/testing/core_utils/configuration/everest_environment_setup.py

```diff
@@ -0,0 +1,255 @@
+
+        return self._EverestEnvironmentTemporaryPaths(
+            ocpp_config_file=ocpp_config_dir / "config.json",
+            ocpp_user_config_file=ocpp_config_dir / "user_config.json",
+            ocpp_database_dir=ocpp_config_dir,
+            certs_dir=certs_dir,
+            ocpp_message_log_directory=ocpp_logs_dir,
+            persistent_store_db_path=persistent_store_dir / "persistent_store.db"
+        )
+
+    def _create_ocpp_module_configuration_visitor(self,
+                                                  temporary_paths: _EverestEnvironmentTemporaryPaths) -> OCPPModuleConfigurationVisitor:
+
+        if self._ocpp_config.ocpp_version == OCPPVersion.ocpp16:
+            ocpp_paths = OCPPModulePaths16(
+                ChargePointConfigPath=str(temporary_paths.ocpp_config_file),
+                MessageLogPath=str(temporary_paths.ocpp_message_log_directory),
+                CertsPath=str(temporary_paths.certs_dir),
+                UserConfigPath=str(temporary_paths.ocpp_user_config_file),
+                DatabasePath=str(temporary_paths.ocpp_database_dir)  # self.temp_ocpp_database_dir.name
+            )
+        elif self._ocpp_config.ocpp_version == OCPPVersion.ocpp201:
+            ocpp_paths = OCPPModulePaths201(
+                ChargePointConfigPath=str(temporary_paths.ocpp_config_file),
+                MessageLogPath=str(temporary_paths.ocpp_message_log_directory),
+                CertsPath=str(temporary_paths.certs_dir),
+                CoreDatabasePath=str(temporary_paths.ocpp_database_dir),
+                DeviceModelDatabasePath=str(temporary_paths.ocpp_database_dir / "device_model_storage.db"),
+            )
+        else:
+            raise ValueError(f"unknown  ocpp version {self._ocpp_config.ocpp_version}")
+
+        occp_module_configuration_helper = OCPPModuleConfigurationVisitor(ocpp_paths=ocpp_paths,
+                                                                          ocpp_module_id=self._ocpp_config.ocpp_module_id,
+                                                                          ocpp_version=self._ocpp_config.ocpp_version)
+
+        return occp_module_configuration_helper
+
+    def _setup_libocpp_configuration(self, source_certs_directory: Path,
+                                     temporary_paths: _EverestEnvironmentTemporaryPaths):
+
+        liboccp_configuration_helper = LibOCPP16ConfigurationHelper() if self._ocpp_config.ocpp_version == OCPPVersion.ocpp16 else LibOCPP201ConfigurationHelper()
+
+        if self._ocpp_config.template_ocpp_config:
+            source_ocpp_config = self._ocpp_config.template_ocpp_config
+        else:
+            source_ocpp_config = self._determine_configured_charge_point_config_path_from_everest_config()
+
+        liboccp_configuration_helper.install_default_ocpp_certificates(
+            source_certs_directory=source_certs_directory,
+            target_certs_directory=temporary_paths.certs_dir)  # Path(self.temp_ocpp_certs_dir.name))
```

Comment:
Do we need these commented paths (here and further down in the file) for anything or are they leftovers?